cluster/gce/configure-vm.sh

```diff
@@ -710,8 +710,7 @@ kind: Config
 users:
 - name: kubelet
   user:
-    client-certificate-data: ${KUBELET_CERT}
-    client-key-data: ${KUBELET_KEY}
```

Comment:
let's figure out why it's unauthorized. what denials are you seeing? is it using the generated kubelet certificate with the system:nodes group?